pkg/bmc/virtualmedia_validation.go

```diff
@@ -0,0 +1,56 @@
+package bmc
+
+import (
+	"github.com/pkg/errors"
+	"github.com/stmcginnis/gofish"
+	"github.com/stmcginnis/gofish/redfish"
+)
+
+// ValidateVirtualMedia returns nil when the BMC has support
+// for VirtualMedia and error when the BMC does not have support.
+func ValidateVirtualMedia(address string, user string, password string, insecure bool) error {
+	bmcConfig := gofish.ClientConfig{
+		Endpoint: address,
+		Username: user,
+		Password: password,
+		Insecure: insecure,
+	}
+
+	client, err := gofish.Connect(bmcConfig)
+	if err != nil {
+		return errors.Wrap(err, "could not connect to redfish BMC.")
+	}
+	defer client.Logout()
+
+	service := client.Service
+
+	managers, err := service.Managers()
+	if err != nil {
+		return errors.Wrap(err, "could not retrive redfish managers.")
```

Comment:
The errors shouldn't have punctuation either https://github.com/golang/go/wiki/CodeReviewComments#error-strings

Reply:
Done

Reply:
Looks like it's still here, do you have some unpushed changes?